generic/secrets/security/detected-npm-registry-auth-token.npmrc

```diff
@@ -1,9 +1,16 @@
 registry="https://registry.npmjs.org/"
 always-auth=true
 package-lock=false
+@myorg:registry=https://somewhere-else.com/myorg
+@another:registry=https://somewhere-else.com/another
+@acme:registry=https://acme.npm/releases
 # Informative
 email=test@example.com
 # Risk
```

Comment:
This is matched as a finding - if this is expected, annotate it with `ruleid: detected-npm-registry-auth-token`

Reply:
probably hits now I've made that regex a little more permissive of spaces 🤔 regardsless, it's a secret and should cause a _hit_ 👍